docs/guides.md

```diff
@@ -523,3 +523,89 @@ Now we enable autostart at the virtual machine level.
 5. Checking the output 
 `# xe vm-param-list uuid=<VM_UUID> | grep other-config`
 
+## Guest UEFI Secure Boot
+
+Enabling UEFI Secure Boot for guests ensures that XCP-ng will ensure that guest VMs only execute trusted binaries.  In practice, these are the binaries released by the operating system (OS) team for the OS in running in the VM (Microsoft Windows, Debian, Alpine, etc.).
+
+### Requirements
+
+* A UEFI guest VM.
+
+### Windows
+
+The steps in this guide are all that are necessary to enable secure boot.  Just follow the instructions provided here.
+
+### Linux
+
+It is required to follow the Linux distribution's instructions for enabling UEFI Secure Boot by installing shim and the distribution's signed binaries.  First follow the instructions from the distribution, then follow the instructions provided here.
+
```

Comment:
As I wrote below, I would move all this under the "Configure the Guest VM" section.

If not, then:
1. I think the Windows and Linux headers should be level `####`, as children of the Requirements header
2. Each header is an anchor, so I usually try to give them simple but unique names (in the page), when I don't forget. "Windows" and "Linux" are a bit too unspecific to me. That's why in my proposal in the comment below I changed them to "Secureboot on Linux VMs" and "Secureboot on Windows VMs"